src/main/java/atsisbot/Ui.java

```diff
@@ -0,0 +1,174 @@
+package atsisbot;
+
+import atsisbot.task.Task;
+import atsisbot.task.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * The Ui class represents the user interface of the ATSISBot application.
+ * It provides methods for reading user commands, printing messages, and
+ * displaying task lists.
+ */
+public class Ui {
+    private static final String line = "____________________________________________________________\n";
+    private static final String welcomeMessage = line + "Hello! I'm atsisbot.AtsisBot\n" + "What can I do for you?\n"
+            + line;
+    private static final String endingMessage = line + "Bye. Hope to see you again soon!\n" + line;
+    private static final String listMessage = "Here are the tasks in your list:\n";
+    private static final String markMessage = "Nice! I've marked this atsisbot.task as done:\n";
+    private static final String unmarkMessage = "OK, I've marked this atsisbot.task as not done yet:\n";
+    private static final String addTaskMessage = "Got it. I've added this atsisbot.task:\n";
+    private static final String deleteTaskMessage = "Noted. I've removed this atsisbot.task:\n";
+    private static final String noDescriptionMessage = "The description of a %s cannot be empty.\n";
+    private static final String invalidDeadlineFormatMessage = "Invalid deadline format. Please use: deadline <description> /by <date>\n";
+    private static final String invalidEventFormatMessage = "Invalid event format. Please use: event <description> /from <date> /to <date>\n";
+    private static final String invalidTaskNumberMessage = "Invalid atsisbot.task number. Please enter a valid atsisbot.task number.\n";
+    private static final String invalidInputFormatMessage = "Invalid input format. Please use the correct format.\n";
+    private static final String unknownCommandMessage = "I'm sorry, but I don't understand that command.";
+    private static final String invalidFindMessage = "No matching tasks found.\n";
+    private static final String findMessage = "Here are the matching tasks in your list:\n";
```

Comment:
Maybe constants should be in screaming snake case?

Reply:
I agree with this! According to our coding standard, **constant names must be all uppercase using underscore to separate words (aka SCREAMING_SNAKE_CASE)**

so maybe your code could look like this:
`private static final String WELCOME_MESSAGE = line + "Hello! I'm atsisbot.AtsisBot\n" + "What can I do for you?\n" + line;`